internal/provider/resource_cloudflare_tiered_cache.go

```diff
@@ -0,0 +1,71 @@
+package provider
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceCloudflareTieredCache() *schema.Resource {
+	return &schema.Resource{
```

Comment:
we'll need to add a `Description` to this for documentation output.

Reply:
Done!